root/components/WorksList.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import WorkListEntry from '../static/scripts/common/components/WorkListEntry';
+
+import SortableTableHeader from './SortableTableHeader';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +checkboxes?: string,
+  +noRatings?: boolean,
```

Comment:
Just having looked at your artists-list PR, it'd be nice to make the prop names consistent between them, so inverting this and making it `showRatings`.

Reply:
I've sent extra commits changing the other existing lists to use positive rather than negative conditions (which seems reasonable, in the vein of doing X ? bar : foo rather than !X ? foo : bar)